packages/workers-playground/src/QuickEditor/QuickEditor.tsx

```diff
@@ -48,11 +48,12 @@ export default function QuickEditor() {
 	useEffect(() => {
 		observeDarkMode(() => setDarkMode(isDarkMode()));
 	}, []);
-	const workerHash = window.location.hash.slice(1);
 
 	const [previewUrl, setPreviewUrl] = React.useState(`/`);
 
-	const [initialWorkerContentHash, setInitialHash] = React.useState(workerHash);
+	const [initialWorkerContentHash, setInitialHash] = React.useState(
+		window.location.hash.slice(1)
+	);
```

Comment:
If this is being read on page load but not being kept up-to-date with changes, we should clear the hash after reading it

Reply:
This is being kept up to date with changes (see `updateWorkerHash` below